python/lsst/dax/imgserv/cli_v1.py

```diff
@@ -157,25 +172,20 @@ def _get_ds(self, req):
         elif image_type == "deepcoadd":
             return W13DeepCoaddDb
 
-    def _get_json_data(self, fp):
-        with open(fp, "r") as f:
-            data = f.read()
-            f.close()
-        return data
-
     def _parse_req(self):
-        req_data = self._get_json_data(self._in_req)
+        with open(self._in_req) as f:
+            req = json.load(f)
+            f.close()
         try:
             if self._validate:
                 # validate the schema
-                schema_data = self._get_json_data(self._schema)
-                schema = json.loads(schema_data)
-                validate(req_data, schema)
+                with open(self._schema) as s:
+                    schema = json.load(s)
+                    s.close()
+                validate(req, schema)
         except ValidationError as e:
-            msg = json.loads({"Validation Error": e.message})
+            msg = json.dumps({"Validation Error": e.message})
             return 1, msg
-        # read in the request (JSON)
-        req = json.loads(req_data)
         return 0, req
 
```

Comment:
Too much whitespace at end of file, can you clean that up? (1 line after last line of code)

Reply:
Deleted blank space as suggested.